satpy/readers/msi_safe.py

```diff
@@ -129,7 +129,7 @@ def get_area_def(self, dsid):
         try:
             from pyproj import CRS
         except ImportError:
-            CRS = None
+            crs = None
```

Comment:
I think it should still be CRS in capitals to match the pyproj name!

Reply:
Done